plugins/inputs/qbittorrent/README.md

```diff
@@ -0,0 +1,69 @@
+# QBittorrent Input Plugin
+
+The qbittorrent plugin will get torrents status.
+
+Compatible qbittorrent API versions must be higher than 2.9.2. 
+Versions lower than 2.9.2 and higher than 2.1.1 are also compatible 
+in principle but have not been tested.
+
+## Global configuration options
+
+## Configuration
+
+```toml @sample.conf
+# Read Qbittorrent status information
+[[inputs.qbittorrent]]
+  ## Url of qbittorrent server
+  # Tls can be:
+  # url = "https://127.0.0.1:8080"
+  url = "http://127.0.0.1:8080"
+
+  ## Credentials for qbittorrent
+  # username = "admin"
+  # password = "admin"
+```
+
+## Metrics 
+
+- torrents
```

Comment:
This section isn't complete, I also see `server_state` and `category` metrics.. Also it isn't clear which of the following items are tags or fields and what their type would be.